stellargraph/core/graph_networkx.py

```diff
@@ -543,6 +543,24 @@ def node_types(self):
                 self._get_node_type(ndata) for n, ndata in self._graph.nodes(data=True)
             }
 
+    def _is_of_edge_type(self, edge, expected_triple):
```

Comment:
I find this method a bit unusual, compared to expressing it with functions that sound more generic:
```python
edge_type_triples_equals(self.edge_type(edge), expected_triple)
```
or something like that, which would basically be splitting this function into two? We already have `def node_type` so would it be good to have `def edge_type` anyway?

Reply:
Yeah, I agree, but this code is disappearing with the new StellarGraph, so I don't think it's worth refactoring in a bigger way like that. I could be convinced otherwise.

Reply:
okay sure 👍